src/components/Btn/Btn.jsx

```diff
@@ -0,0 +1,27 @@
+/*
+We're constantly improving the code you see. 
+Please share your feedback here: https://form.asana.com/?k=uvp-HPgd3_hyoXRBw1IcNg&d=1152665201300829
+*/
+
+import PropTypes from "prop-types";
```

Comment:
[PropTypes are deprecated](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#removed-proptypes-and-defaultprops). Please use TypeScript types instead. The Redback UI repository has some examples you may find helpful.